mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -49,6 +49,9 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     density_comp: np.ndarray default None
         The density compensation for kspace data in case it exists and we
         use density compensated adjoint for Smap estimation
+    window_fun: "Hann(ing)", "Hamming", or a callable, default None
```

Comment:
Please explain what the callable should take as input and what it should return.

Reply:
Also, is default of None a good idea? Maybe we move the default to use this and not cause any ringing artifacts?
@paquiteau Ill let you take the call on which window is best use case for us.

Reply:
I left it to None to not change expected results downstream, It will required more testing to be sure that using a window function is better in *every* cases, (eg 3D). Furthermore, if the windowing reduces the ringing, it does not create perfect Smaps either. I see this windowing possibility as an entry point for more complex Smaps estimations.